mcstatus/querier.py

```diff
@@ -1,11 +1,14 @@
 from __future__ import annotations
 
 import random
-import struct
 import re
+import struct
 from typing import List
 
-from mcstatus.protocol.connection import Connection, UDPAsyncSocketConnection, UDPSocketConnection
+from mcstatus.protocol.connection import Connection, TYPE_CHECKING, UDPAsyncSocketConnection, UDPSocketConnection
```

Comment:
Should `TYPE_CHECKING` be imported from `typing` instead?

Reply:
oh, I suppose my editor auto-imported that from a wrong location and I didn't notice, interesting that it did that, yeah it should certainly come from `typing`

Reply:
Solved in 7735555